src/components/15Years/15YearsHeader.vue

```diff
@@ -302,14 +304,41 @@ export default {
 			}
 			this.advanceTimeout = null;
 		},
-	},
-	inject: ['apollo'],
-	apollo: {
-		query: countryQuery,
-		preFetch: true,
-		result({ data }) {
-			console.log('Fetched data from graphql~!', data);
+		fetchLiveCountryData() {
+			const query = this.apollo.query({
+				query: countryQuery,
+			});
+			query.then(({ data }) => {
+				// flatten country object and pass to list builder
+				const countries = data.lend?.countryFacets.map(obj => obj.country);
```

Comment:
All these changes scan and make sense to me, but this line throws an error for me in Storybook; `data` is coming in as undefined. Do I need to be running a separate process for graphql?

Reply:
ahh yes my apologies... I didn't mock out any data for storybook...

Reply:
I'll work on adding that in a few